src/apps/testapps/testH3Index.c

```diff
@@ -47,6 +47,20 @@ TEST(h3IsValidDigits) {
              "h3IsValid failed on invalid unused digits");
 }
 
+TEST(h3IsValidBaseCell) {
```

Comment:
I think we're losing a test that an index with an invalid base cell (such as 127) is invalid.

Reply:
Oh? I'll look for it and include it in here. I satisfied myself with reaching 100% code coverage last night.

Reply:
Done.